.circleci/config.yml

```diff
@@ -610,11 +585,7 @@ workflows:
       - deploy_templates:
           requires:
             - build-push-3scale-images
-      - install-operator
-      - run-operator-e2e-test:
-          requires:
-            - install-operator
-            - build-push-3scale-images
+      - build-operator-image-only
```

Comment:
I would also add to the `operator` workflow to have the image built for each PR

Reply:
Changed